Docs/Configuration.tex

```diff
@@ -2998,66 +3018,68 @@ \subsection{Boot Properties}\label{miscbootprops}
     \item Entry is system (e.g. \texttt{Reset NVRAM}).
   \end{itemize}
 
-  To see all entries picker menu needs to be reloaded in extended mode by pressing
-  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance
-  for multidisk systems.
-
+  To display all entries, the picker menu can be reloaded into ``Extended Mode'' by pressing the
+  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance on multidisk systems.
```

Comment:
I am not a native English speaker, but I still think that `multi` and `disk` should be connected with a dash. i.e. `multi-disk`.